docs/Development-Guides/Rate-Limits.md

```diff
@@ -29,3 +22,166 @@ Product | Max requests initiated per minute
 `employment` | 4
 `payment` | 2
 `pay-statement` | 2
+
+## Application Rate Limits
+
+Similarly, multiple access tokens can be created from a single Finch application as more employers are connected. A Finch application has its own rate limits separate from the access token rate limits. You can think of an application like a development environment or application stage. Finch will assign three types of applications: `sandbox`, `development`, and `production`. Each will have their own `client_id` which represents the application.
```

Comment:
I'd omit the part about the kinds/types of applications, seems irrelevant to the rate limiting. I'd hope a developer knows their Client ID. I think that's a fairly common pattern.

Reply:
Unfortunately, some developers need this explained, so I'll keep it unless anyone else opposes it.

Reply:
Maybe it should go in its own document/page explaining the Client ID differences?